tests/unit/api/v1/test_status.py

```diff
@@ -45,9 +47,18 @@ class HealthApiTestCase(test.TestCase):
 
     def setUp(self):
         super(HealthApiTestCase, self).setUp()
-        self.health_config = {"health": {
-            "endpoint": "http://dummy.example.org/api/health"
-        }}
+        self.health_config = {
+            "health": {
+                "endpoint": "http://dummy.example.org/api/health"
+            }
+        }
+
+        self.old_USE_FAKE_DATA = fake_api_base.USE_FAKE_DATA
+        fake_api_base.USE_FAKE_DATA = False
```

Comment:
This is not clear to me, does this mean that Fake API is not tested?

What do you think about approach of having 2 test classes for each module:

class ApiTestCase(test.TestCase):
     <tests for real API>

class FakeApiTestCase(test.TestCase):
     <tests for fake API>